docs/submission/submitting-molecular-data.md

```diff
@@ -60,39 +60,39 @@ storage.url=https://score.qa.argo.cancercollaboratory.org
 
 ### Understanding Song payload fields
 
-Song accepts data in JSON format, which is validated against standard JSON Schema to ensure data quality. The first step of submitting sequencing data to ARGO is to prepare the Song metadata payloads conforming to the most recent JSON schema that has been defined by the DCC. The latest `sequencing_experiment` JSON Schema can be found in the ARGO [github repository](https://github.com/icgc-argo/argo-metadata-schemas/blob/master/schemas/sequencing_experiment.json).
+Song accepts metadata in JSON format (also referred as a `Song payload`), which is validated against standard JSON Schema to ensure data quality. The first step of submitting sequencing data to ARGO is to prepare the Song metadata payloads conforming to the most recent JSON schema that has been defined by the DCC. The latest `sequencing_experiment` JSON Schema can be found in the ARGO [github repository](https://github.com/icgc-argo/argo-metadata-schemas/blob/master/schemas/sequencing_experiment.json).
 
 The data fields can be broken down into four main sections: `experiment`, `sample`, `read groups` and `files`.
 
 > Note: \*\* denotes a required field. These must be provided as part of the metadata payload or it will immediately fail validation upon submission.
 
-#### **Experiment:** The experiment section contains details that are relevant to the experimental requirements imposed during sequencing.
+**Experiment:** The experiment section contains details that are relevant to the experimental requirements imposed during sequencing.
 
 - `**studyId`: Corresponds to your ARGO `Program ID`. This is the unique id that is assigned to your program. If you have logged into the ARGO Data Platform, this is the Program Id that you see in the Program Services area. For example, PACA-CA is a Program ID.

```

Comment:
`studyId` is not under `experiment` section.

Reply:
added a main body sectoin